t/lib/OpenQA/SeleniumTest.pm

```diff
@@ -324,7 +324,7 @@ sub element_not_present {
 sub wait_until {
     my ($check_function, $check_description, $timeout, $check_interval) = @_;
     $timeout        //= 10;
-    $check_interval //= 0.2;
+    $check_interval //= 1;
```

Comment:
this means we waste more time? Can we keep the old value?

Reply:
Perhaps it's more constructive to ask for the motivation here: why is it helpful to increase the default interval?

I don't expect this to translate to general slowdowns, though, since the timeout is the same.

Reply:
It is a separate commit with its own commit message: https://github.com/os-autoinst/openQA/pull/2357/commits/313790d8be001ce607a28ecb780fc8799abf1916

Reply:
So no, this will decrease the time wasted when the element never shows up. Otherwise it shouldn't make a difference.

Reply:
Sorry. Didn't want to be unconstructive. The change *seemed* to be a temporary debugging change. I missed the explicit commit. All is good :)

Reply:
Well ok, not *all* is well, the commit message subject line describes the obvious, not the effect and also has a typo. Please keep in mind that we generate the changelog from the git commit message subject lines and people do read the changelog